test/dao/fetchCatalystStatuses.test.ts

```diff
@@ -2,37 +2,21 @@ import sinon from 'sinon'
 import { expect } from 'chai'
 import { fetchCatalystStatuses } from 'shared/dao'
 import * as ping from 'shared/dao/utils/ping'
-import { Candidate, PingResult } from 'shared/dao/types'
-
-const pingResult = (maxUsers?: number): PingResult => ({
-  status: 0,
-  elapsed: 309,
-  result: {
-    name: 'loki',
-    version: '1.0.0',
-    usersCount: 59,
-    maxUsers: maxUsers || 1000
-  }
-})
-
-const NODES = [
-  {
-    domain: 'peer.decentraland.org'
-  },
-  {
-    domain: 'peer-ec1.decentraland.org'
-  }
-]
+import { Candidate } from 'shared/dao/types'
 
 const EXPECTED: Candidate = {
   protocol: 'v2',
   catalystName: 'loki',
   domain: 'peer.decentraland.org',
   elapsed: 309,
-  maxUsers: 1000,
+  maxUsers: -1,
   status: 0,
   usersCount: 59,
-  usersParcels: undefined
+  usersParcels: [
+    [1, 1],
+    [1, 1],
```

Comment:
shouldn't the parcels be different from each other?

Reply:
not here, in the about response yes, but here it expects a parcel per user